tests/integration/testdata/buildcmd/Java/gradle-kotlin/build.gradle.kts

```diff
@@ -8,5 +8,5 @@ repositories {
 
 dependencies {
     implementation("software.amazon.awssdk:annotations:2.1.0")
-	compile("com.amazonaws:aws-lambda-java-core:1.1.0")
+    implementation("com.amazonaws:aws-lambda-java-core:1.1.0")
```

Comment:
Is `implementation` supported in the earlier versions of Gradle?

Reply:
implementation and api were added back in gradle 3.0.0 for replacing compile.
This change is also only related our test code and it will not affect the production.